ConcurrentPercolator/ConcurrentPercolator.tla

```diff
@@ -251,10 +254,9 @@ Prewrite(c) ==
   /\ client_state[c] = "prewriting"
   /\ IF client_key[c].pending = {}
      THEN  \* all keys have been pre-written
-       /\ next_ts' = next_ts + 1
        /\ client_state' = [client_state EXCEPT ![c] = "committing"]
-       /\ client_ts' = [client_ts EXCEPT ![c].commit_ts = next_ts']
-       /\ UNCHANGED <<key_vars, client_key>>
+       /\ client_ts' = [client_ts EXCEPT ![c].commit_ts = client_ts[c].commit_ts + 1]
```

Comment:
Comment something like, `client_ts[c].commit_ts` is set to `start_ts` at first, then updated at every prewrite, and finally increased by 1.